website/source/docs/providers/google/index.html.markdown

```diff
@@ -39,17 +39,28 @@ The following keys can be used to configure the provider.
   retrieving this file are below. Credentials may be blank if you are running
   Terraform from a GCE instance with a properly-configured [Compute Engine
   Service Account](https://cloud.google.com/compute/docs/authentication). This
-  can also be specified with the `GOOGLE_CREDENTIALS` or `GOOGLE_CLOUD_KEYFILE_JSON`
-  shell environment variable, containing the contents of the credentials file.
+  can also be specified using any of the following environment variables
+  (listed in order of precedence):
```

Comment:
Could you point to the docs where you found these requirements here? It will make it clearer why Terraform allows you to configure your credentials so many different ways.